src/index.html

```diff
@@ -15,8 +15,97 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a class="header_logo_link">
+          <img
+            class="header_logo"
+            src="images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+
+        <nav class="header_nav">
+          <ul class="header_list">
+            <li class="header_item">
+              <a
+                href="#"
```

Comment:
The `href` attribute is currently set to `#`, which is a placeholder. Consider updating it to point to the actual URL or section of the page this link should navigate to.